app/src/main/java/fr/free/nrw/commons/modifications/ModificationsContentProvider.java

```diff
@@ -38,11 +40,11 @@ public static Uri uriForId(int id) {
         return Uri.parse(BASE_URI.toString() + "/" + id);
     }
 
-    @Inject DBOpenHelper dbOpenHelper;
+    @Inject Lazy<DBOpenHelper> dbOpenHelper;
```

Comment:
Rename for clarity?  See comments on `CategoryContentProvider`

Reply:
No longer using `Lazy`